akp/resource_akp_cluster_schema.go

```diff
@@ -619,3 +624,16 @@ func getArgoCDNotificationsSettingsAttributes() map[string]schema.Attribute {
 		},
 	}
 }
+
+func getDirectClusterSpecAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"cluster_type": schema.StringAttribute{
```

Comment:
Do we need to make `cluster_type` and `kargo_instance_id` computed like other fields?